src/features/listings/components/ListingPage/Comments.tsx

```diff
@@ -8,25 +9,26 @@ import {
   VStack,
 } from '@chakra-ui/react';
 import axios from 'axios';
-import { useEffect, useState } from 'react';
-import { GoCommentDiscussion } from 'react-icons/go';
+import Image from 'next/image';
+import { useEffect, useRef, useState } from 'react';
 
 import { LoginWrapper } from '@/components/LoginWrapper';
 import { ErrorInfo } from '@/components/shared/ErrorInfo';
 import { Loading } from '@/components/shared/Loading';
 import { UserAvatar } from '@/components/shared/UserAvatar';
 import type { Comment } from '@/interface/comments';
 import { userStore } from '@/store/user';
-import { dayjs } from '@/utils/dayjs';
-import { getURL } from '@/utils/validUrl';
 
+import { autoResize } from '../../utils';
```

Comment:
we already have an autoresize component: 
https://github.com/SuperteamDAO/earn/blob/main/src/components/shared/autosize-textarea.tsx